common/reflectivity/reflectivity.cpp

```diff
@@ -1,9 +1,13 @@
 // License: Apache 2.0. See LICENSE file in root directory.
 // Copyright(c) 2020 Intel Corporation. All Rights Reserved.
 
-#include <numeric>
 #include "reflectivity.h"
-#include <types.h>
+#include <vector>
```

Comment:
BTW, Reflectivity & MUR can both be deleted totally., not used.
Both are L515 only and was origin by L515 reflectivity issues

Reply:
Yes I know, we spoke about it and, at least for now, I decided to leave them alone. Let's discuss offline.